Instance,Runtime (seconds),Feasible,Profit,Error
lrc53B.txt,0.14,No,-inf,"[Error running script: Command '['python3', 'main.py', '--instance_path', '/home/akkermanfr/competition_master/Instances/lrc53B.txt']' returned non-zero exit status 2.. Details: usage: main.py [-h]
main.py: error: unrecognized arguments: --instance_path /home/akkermanfr/competition_master/Instances/lrc53B.txt
]"
lrc103B.txt,0.14,No,-inf,"[Error running script: Command '['python3', 'main.py', '--instance_path', '/home/akkermanfr/competition_master/Instances/lrc103B.txt']' returned non-zero exit status 2.. Details: usage: main.py [-h]
main.py: error: unrecognized arguments: --instance_path /home/akkermanfr/competition_master/Instances/lrc103B.txt
]"
lrc303B.txt,0.14,No,-inf,"[Error running script: Command '['python3', 'main.py', '--instance_path', '/home/akkermanfr/competition_master/Instances/lrc303B.txt']' returned non-zero exit status 2.. Details: usage: main.py [-h]
main.py: error: unrecognized arguments: --instance_path /home/akkermanfr/competition_master/Instances/lrc303B.txt
]"
